build/llvm.yml

```diff
@@ -0,0 +1,212 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+trigger: none
+
+variables:
+  CCACHE_DIR: $(Pipeline.Workspace)/ccache
+  AQ_LLVM_PACKAGE_GIT_VERSION: ""
```

Comment:
If `AQ` is for Azure Quantum, we're actually using `AZURE_QUANTUM` for variable names. Wondering if this really needs prefixing as it's only used by our internal build.